tests/test_network.py

```diff
@@ -241,3 +241,42 @@ async def test_network_network_response_completed_event_emitted(
             }
         }
     }
+
+
+@pytest.mark.asyncio
+async def test_network_bad_ssl(websocket, context_id):
+    BAD_SSL_URL = "https://expired.badssl.com/"
```

Comment:
We have a bunch of TODOs to make the tests work offline. Does it make sense to add one here as well?

Reply:
added